ice/recipes/elicit/search.py

```diff
@@ -25,10 +25,13 @@ def make_request_body(
 async def elicit_search(
     question: str = "What is the effect of creatine on cognition?",
     num_papers: int = 4,
+    endpoint: str = ELICIT_SEARCH_ENDPOINT,
```

Comment:
Let's get rid of the constant and inline it here, especially since it isn't really correct (ideally this recipe would default to the result of https://elicit.org/api/backend, but that's more complicated)